app/src/main/java/com/hajduk/systems/prepareordermailing/view/ConfirmData.kt

```diff
@@ -0,0 +1,64 @@
+package com.hajduk.systems.prepareordermailing.view
+
+import android.content.Intent
+import android.os.Bundle
+import android.widget.Button
+import android.widget.Toast
+import android.widget.Toast.LENGTH_LONG
+import androidx.appcompat.app.AppCompatActivity
+import com.hajduk.systems.prepareordermailing.R
+import com.hajduk.systems.prepareordermailing.adapter.woocommerce.WooCommerceClient
+import com.hajduk.systems.prepareordermailing.adapter.woocommerce.model.CustomerDto
+import com.hajduk.systems.prepareordermailing.adapter.woocommerce.model.OrderDto
+import com.hajduk.systems.prepareordermailing.holder.ApplicationDataHolder
+import kotlinx.android.synthetic.main.activity_confirm_data.*
+
+class ConfirmData : AppCompatActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_confirm_data)
+        retrieveAndDisplayData()
+
+        findViewById<Button>(R.id.dataRejectButton).setOnClickListener { startActivity(Intent(this, ScanBarCode::class.java)) }
+        findViewById<Button>(R.id.dataConfirmButton).setOnClickListener { startActivity(Intent(this, TakePhoto::class.java)) }
+    }
+
+    private fun retrieveAndDisplayData() {
+        val wooCommerceClient =
+            WooCommerceClient("http://localhost", "http://10.0.2.2", "ck_132211fbf3952da2e259ac134d456f59c15ae8d3", "cs_b850bfa01ca6eef3c4ae1fb73b9d781524651cc4")
+        retrieveOrderData(wooCommerceClient, ApplicationDataHolder.instance.demandOrderId())
+    }
+
+    private fun retrieveOrderData(wooCommerceClient: WooCommerceClient, orderId: Int) {
+        wooCommerceClient.getOrder(
```

Comment:
miałbym trochę inną koncepcję na to - zaproponuję w przyszłym tyg i ocenimy sobie